introduction.md

```diff
@@ -0,0 +1 @@
+Hello! I am here to teach people about Github.
```

Comment:
I am not receiving the new_rules branch.  I can't seem to get it added to my repository.  I added @codeschool-kiddo as a collaborator, but the Deadly Skills list was the last thing I was asked to do.